public/checkout.js

```diff
@@ -309,20 +329,11 @@ var packeteryLoadCheckout = function( $, settings ) {
 			}
 
 			var clearSavedData = false;
-			if ( hasPickupPoints( initialCarrierRateId ) ) {
-				if ( initialDestinationAddress.country !== destinationAddress.country ) {
-					clearSavedData = true;
-				}
-			} else if ( hasHomeDelivery( initialCarrierRateId ) ) {
-				if (
-					initialDestinationAddress.country !== destinationAddress.country ||
-					initialDestinationAddress.street !== destinationAddress.street ||
-					initialDestinationAddress.city !== destinationAddress.city ||
-					initialDestinationAddress.postCode !== destinationAddress.postCode
-				) {
-					clearSavedData = true;
-				}
-			} else if ( hasCarDelivery( initialCarrierRateId ) ) {
+			if (
```

Comment:
K ticketu je třeba napsat, že to mění chování a proč.

Reply:
Provedu!